stix_shifter_modules/aws_athena/stix_translation/json/stix_2_1/ocsf_from_stix_map.json

```diff
@@ -0,0 +1,820 @@
+      "extensions.'x-ocsf-file-ext'.version": [
+        "file.version"
+      ],
+      "name": [
+        "file.name"
+      ],
+      "size": [
+        "file.size"
+      ]
+    }
+  },
+  "ipv4-addr": {
+    "fields": {
+      "value": [
+        "dst_endpoint.ip",
+        "src_endpoint.ip"
+      ]
+    }
+  },
+  "ipv6-addr": {
+    "fields": {
+      "value": [
+        "dst_endpoint.ip",
+        "src_endpoint.ip"
+      ]
+    }
+  },
+  "network-traffic": {
+    "fields": {
+      "dst_byte_count": [
+        "traffic.bytes_in"
+      ],
+      "dst_packets": [
+        "traffic.packets_in"
+      ],
+      "dst_port": [
+        "dst_endpoint.port"
+      ],
+      "dst_ref.value": [
+        "dst_endpoint.ip"
+      ],
+      "extensions.'x-network-ext'.boundary": [
+        "connection_info.boundary"
+      ],
+      "extensions.'x-network-ext'.boundary_id": [
+        "connection_info.boundary_id"
+      ],
+      "extensions.'x-network-ext'.bytes": [
+        "traffic.bytes"
+      ],
+      "extensions.'x-network-ext'.direction": [
+        "connection_info.direction"
+      ],
+      "extensions.'x-network-ext'.direction_id": [
+        "connection_info.direction_id"
+      ],
+      "extensions.'x-network-ext'.packets": [
+        "traffic.packets"
+      ],
+      "extensions.'x-network-ext'.protocol_ver": [
+        "connection_info.protocol_ver"
+      ],
+      "extensions.'x-network-ext'.tcp_flags": [
+        "connection_info.tcp_flags"
+      ],
+      "protocols[*]": [
+        "connection_info.protocol_num"
+      ],
+      "src_byte_count": [
+        "traffic.bytes_out"
+      ],
+      "src_packets": [
+        "traffic.packets_out"
+      ],
+      "src_port": [
+        "src_endpoint.port"
+      ],
+      "src_ref.value": [
+        "src_endpoint.ip"
+      ]
+    }
+  },
+  "process": {
+    "fields": {
+      "binary_ref.name": [
+        "file.name"
+      ],
+      "command_line": [
+        "process.cmd_line"
+      ],
+      "created": [
+        "process.created_time"
+      ],
+      "extensions.'x-ocsf-process-ext'.integrity": [
+        "process.integrity"
+      ],
+      "extensions.'x-ocsf-process-ext'.integrity_id": [
+        "process.integrity_id"
+      ],
+      "extensions.'x-ocsf-process-ext'.is_system": [
+        "process.is_system"
+      ],
+      "extensions.'x-ocsf-process-ext'.lineage": [
+        "process.lineage"
+      ],
+      "extensions.'x-ocsf-process-ext'.loaded_modules": [
+        "process.sandbox"
+      ],
+      "extensions.'x-ocsf-process-ext'.terminated_time": [
+        "terminated_time"
+      ],
+      "extensions.'x-ocsf-process-ext'.tid": [
+        "process.tid"
+      ],
+      "x_unique_id": [
+        "process.uid"
+      ],
+      "extensions.'x-ocsf-process-ext'.xattributes": [
+        "process.xattributes"
+      ],
+      "mime_type": [
+        "mime_type"
+      ],
+      "modified": [
+        "process.modified_time"
+      ],
+      "name": [
```

Comment:
The name property on process was removed in 2.1. This would have to be changed to `x_name`

Reply:
updated